### PR TITLE
Add in airflow, dbt Cloud Provider into official ways to run dbt in production

### PR DESCRIPTION
**Pull request by sungchun12**
## Description & motivation

Adding in a better way to interface airflow and dbt Cloud based on this pull request: https://github.com/apache/airflow/pull/20998

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

File: website/docs/docs/running-a-dbt-project/running-dbt-in-production.md
Comment (mik-laj) [inline website/docs/docs/running-a-dbt-project/running-dbt-in-production.md]:
```suggestion
* Invoking dbt through the [DbtCloudRunJobOperator](https://airflow.apache.org/docs/apache-airflow-providers-dbt-cloud/stable/operators.html).
```

Comment (sungchun12) [reply website/docs/docs/running-a-dbt-project/running-dbt-in-production.md]: I also recommend adding a link for the astronomer registry: https://registry.astronomer.io/providers/dbt-cloud?type=Sensors&utm_campaign=Monthly%20Product%20Updates&utm_medium=email&_hsmi=208603877&utm_content=208603877&utm_source=hs_email

**Comment by sungchun12** [comment]
@asteichen will officially own this PR

**Comment by sungchun12** [comment]
image
Have this official airflow provider be the first option.

Include a screenshot of Airflow DAG working AND dbt Cloud job that references the airflow DAG.

File: website/docs/docs/running-a-dbt-project/running-dbt-in-production.md
Comment (sungchun12) [inline website/docs/docs/running-a-dbt-project/running-dbt-in-production.md]:
```suggestion
* Invoking dbt through the [DbtCloudRunJobOperator](https://airflow.apache.org/docs/apache-airflow-providers-dbt-cloud/stable/operators.html). See the [Astronomer Registry](https://registry.astronomer.io/providers/dbt-cloud?type=Operators) to see an [example airflow DAG](https://registry.astronomer.io/dags/example-dbt-cloud) to get started!
```

File: website/docs/docs/running-a-dbt-project/running-dbt-in-production.md
Comment (sungchun12) [inline website/docs/docs/running-a-dbt-project/running-dbt-in-production.md:40]: The images look great!

**Review by sungchun12 — COMMENTED** [review]
Check out my suggestions!

**Comment by sungchun12** [comment]
@runleonarun any updates on this?

**Comment by sungchun12** [comment]
@dbt-labs/oss-maintainers Any review for this?

**Review by krevitt — CHANGES_REQUESTED** [review]
Looks like the [running dbt in production](https://docs.getdbt.com/docs/running-a-dbt-project/running-dbt-in-production#using-airflow) page was updated in another PR to include the provider + a link to the Astronomer guide.  

Want to resolve merge conflicts there @sungchun12?  Looks like this PR can resolve to adding the images.

**Comment by sungchun12** [comment]
@asteichen I'll let you take the lead to resolve conflicts!

**Comment by runleonarun** [comment]
@sungchun12 sorry it's taken so long to get to this. I'm trying to get my GitHub notifications under control. Once those conflicts are resolved, I can approve!

**Comment by asteichen** [comment]
@krevitt @runleonarun  I added the images, made the dbt Cloud operator the first bullet, and combined that description a bit.

**Review by krevitt — APPROVED** [review]
LGTM!

**Comment by sungchun12** [comment]
@asteichen you have the honor of officially merging this branch!

**Pull request merged**
